### DISCUSSION TITLE
Данные для обучения

### DISCUSSION THREAD

**Discussion by ValiullinAlbert** [ispras/dedoc#446] [General]
Добрый день! Можно ли как-то получить доступ к данным, на которых вы обучались, чтобы попытаться дообучить модель, в том числе используя свои собственные данные?

**Comment by NastyBoget** [comment]
Добрый день! Данные, на которых обучены текущие классификаторы есть на облаке, ссылки можно посмотреть в скриптах обучения `scripts/train/train_{something}.py` при инициализации `XGBoostLineClassifierTrainer`, ссылка на данные - значение аргумента `data_url`. Например, https://at.ispras.ru/owncloud/index.php/s/TNFXFQQKICa5oGy/download в `scripts/train/train_diploma_line_classifier.py`.

Также мы переразметили данные, но пока не переобучали на них классификаторы, новые данные находятся в разделе datasets на huggingface: https://huggingface.co/dedoc

**Reply by ValiullinAlbert** [reply]
Спасибо!
